SlevomatCodingStandard/Sniffs/Complexity/CognitiveSniff.php

```diff
@@ -101,8 +102,14 @@ class CognitiveSniff implements Sniff
 		T_BREAK => T_BREAK,
 	];
 
-	/** @var int */
-	public $maxComplexity = 5;
+	/** @var ?int maximum allowed complexity (deprecated) */
```

Comment:
Please use `@deprecated`

Reply:
should a E_USER_DEPRECATED  be triggered if used ?

Reply:
No, `@deprecated` annotation is enough. It's primary for me.